pytorch/inference/docker/1.13/py3/cu117/Dockerfile.gpu

```diff
@@ -0,0 +1,314 @@
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.9.13
+ARG MAMBA_VERSION=22.9.0-1
```

Comment:
why do Mamba versions differ btwn CPU and GPU? 

Looks like we've been using 4.12 for a long time, @RadhikaB-97 or @jeet4320 do you see any reason to update/not-update mamba version?

Reply:
I Updated CPU to match GPU. Was so focused on insuring the GPU was up to date and working, guess I missed that on on the GPU. This would be a good example where the common items between the CPU and GPU images could be a "base".